src/Parser/grammar.pp

```diff
@@ -0,0 +1,90 @@
+%skip   space               [\x20\x09\x0a\x0d]+
+%skip   doc_                [/**]
+%skip   _doc                [*/]
+%skip   star                [*]
+
+%token  at                  @                           -> annot
+%token  text                [^@].*
+
+%token  annot:identifier    [\\a-zA-Z_][\\a-zA-Z0-9_]*  -> values
```

Comment:
sorry, a star is missing in my suggestion. should be ``\\?[a-zA-Z_][a-zA-Z0-9_]*(\\[a-zA-Z_][\\a-zA-Z0-9_])*``